src/pipeline_factory.cpp

```diff
@@ -25,7 +25,6 @@ Status PipelineFactory::createDefinition(const std::string& pipelineName,
     const pipeline_connections_t& connections,
     ModelManager& manager) {
     if (definitionExists(pipelineName)) {
-        SPDLOG_LOGGER_WARN(modelmanager_logger, "Two pipelines with the same name: {} defined in config file. Ignoring the second definition", pipelineName);
```

Comment:
I think we can leave this log here since we are not supposed to enter here anyway. If we find such line in logs it means that OVMS logic is broken somewhere else.